enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/components/SdkSaveQuestionForm.tsx

```diff
@@ -34,7 +34,7 @@ export const SdkSaveQuestionForm = ({ onCancel }: SdkSaveQuestionFormProps) => {
       targetCollection={id}
     >
       <Stack p="md">
-        <Title>
+        <Title order={2}>
```

Comment:
@oisincoveney can you help us think through the consequences of this for theming?

This PR makes h1 32px, where before it was 24px, so moving this from h1 (the default) to h2 keeps it the same size with the metabase theme. However, if someone has overridden our theme, we're going to pick up their h2 theme rather than their h1 theme. BUT, if we leave it, it's going to be too big for anyone who hasn't themed it 🥴